packages/story-editor/src/app/media/uploadQueue/index.ts

```diff
@@ -476,65 +529,69 @@ function useMediaUploadQueue() {
       /**
        * Uploads a single pending item.
        *
-       * @param {Object} item Queue item.
-       * @param {File} item.file File object.
-       * @param {Object} item.additionalData Additional Data object.
+       * @param item Queue item.
+       * @param item.file File object.
+       * @param item.additionalData Additional Data object.
```

Comment:
Just remove these JSDoc things - the TS types express the contents much better anyway.